src/WCSimSteppingAction.cc

```diff
@@ -100,7 +100,7 @@ G4int WCSimSteppingAction::G4ThreeVectorToWireTime(G4ThreeVector *pos3d,
 } 
 
 
-void WCSimSteppingAction::Distortion(G4double x,G4double y)
+void WCSimSteppingAction::Distortion(G4double /*x*/,G4double /*y*/)
```

Comment:
Would you mind explaining in a bit more detail why these changes should be taken out of this PR? It's likely to be an hour or twos work, for (from what I can see at the moment) no gain - whether the changes are merged here or later they'll still be in the develop branch & the next release.